### PR TITLE
text recognition

### PR DESCRIPTION
**Pull request by rbelmega**

### CODE REVIEW

File: src/components/Camera/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import styles from "./styles";
+
+const PICTURE_OPTIONS = {
+  quality: 1,
+  fixOrientation: true,
+  forceUpOrientation: true
+};
+
+export default class App extends React.Component {
+  state = {};
```
Comment (vasinkevych) [inline src/components/Camera/index.js]: flow? later? :)

File: src/components/Camera/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    const data = await this.camera.takePictureAsync(PICTURE_OPTIONS);
+    this.props.snap(data);
+  };
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <RNCamera
+          ref={ref => (this.camera = ref)}
```
Comment (vasinkevych) [inline src/components/Camera/index.js]: is it possible to use new refapi? c
class property: camera = createRef()
here just: ref={this.camera}

File: src/components/Camera/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  forceUpOrientation: true
+};
+
+export default class App extends React.Component {
+  state = {};
+
+  flip = () => {
+    this.setState({
+      type:
+        this.state.type === RNCamera.Constants.Type.back
```
Comment (vasinkevych) [inline src/components/Camera/index.js]: https://medium.com/@WoTzap/%D1%84%D1%83%D0%BD%D0%BA%D1%86%D0%B8%D0%BE%D0%BD%D0%B0%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9-setstate-%D1%8D%D1%82%D0%BE-%D0%B1%D1%83%D0%B4%D1%83%D1%89%D0%B5%D0%B5-react-d262be6a6c02
functional setState is preferred but not required

File: src/components/Camera/index.js (hunk 1 above)
Comment (rbelmega) [reply src/components/Camera/index.js]: not this time :)

**Comment by vasinkevych** [comment]
👍

**Pull request merged**